src/components/app/Home.js

```diff
@@ -251,7 +251,7 @@ class Home extends PureComponent {
             <InstallButton name="Gecko Profiler" xpiURL={LEGACY_ADDON_URL}>
               Gecko Profiler Add-on
             </InstallButton>. Then use the button added to the browser, or use
-            the following shortcuts to record a profile. The button's icon is
+            the following shortcuts to record a profile. The button’s icon is
```

Comment:
I guess it'll also fix GitHub's formatting issues too. I'm not sure how to enter a curly quote though.